#### Summary
On this page we are linking to various material that have to do with our investigations of the origin and evolution of SARS-CoV-2, the virus responsible for the COVID-19 pandemic.

#### Key Papers
[The Proximal Origin of SARS-CoV-2 - Nature Medicine, 2020](https://www.sciencedirect.com/science/article/pii/S0092867421009910)

[The Origins of SARS-CoV-2: A Critical Review - Cell, 2021](https://www.sciencedirect.com/science/article/pii/S0092867421009910)

#### Paper - The Proximal Origin of SARS-CoV-2
This short peer-reviewed article in [Nature Medicine](https://www.nature.com/articles/s41591-020-0820-9) summarizes our scientific investigations into the proximal origins of SARS-CoV-2. Due to constraints on article format, many references could not be included, and experimental details were limited. We deeply investigated both natural and artificial scenarios for the proximal origin of SARS-CoV-2. Our analyses strongly support a natural origin of the virus and all the data are fully consistent with evolutionary processes having led to the emergence of SARS-CoV-2 in the human population.

This study was motivated by high-level concerns about potential lab origins of the virus, decade-long research in Wuhan on bat coronaviruses, and, of course, scientific curiosity of how this virus likely came to be. There was also a separate call from the [White House OSTP](https://www.nationalacademies.org/news/2020/02/national-academies-provide-rapid-response-to-white-house-on-coronavirus-data-needs) to understand this question better, which was initiated in response to our early analyses.

The two main features that caught our eye as we started these analyses were the receptor binding domain (RBD) in SARS-CoV-2 and the polybasic (furin) cleavage site between the S1 and S2 subunits (and associated predicted O-linked glycans). 

The RBD appeared to be a high affinity binder of the human ACE2 receptor (based on structural modeling we performed - but we now know this to be [true](https://science.sciencemag.org/content/367/6483/1260)), which was puzzling when compared to SARS-CoV and other coronaviruses. This particular RBD had not previously been observed in nature, however, as we were conducting our analyses, several groups [reported](https://pubmed.ncbi.nlm.nih.gov/32218527/) that this  RBD can also be found in coronaviruses sampled in pangolins, conclusively proving that the SARS-CoV-2 RBD is present in nature and the result of natural evolution. It's important to understand that finding it in a pangolin coronavirus, does not necessarily mean that SARS-CoV-2 (or the RBD) came from pangolins - we can only sample a very tiny fraction of the entire virosphere out there, so this RBD may also be present in all sorts of other (unsampled) coronaviruses, [including in bats](https://pubmed.ncbi.nlm.nih.gov/32724171/). Update: this RBD has now also been found in [bat CoVs in Laos](https://www.researchsquare.com/article/rs-871965/v1). 

When we performed our analyses, the polybasic cleavage site we identified was predicted, but we now know [it's functional](https://pubmed.ncbi.nlm.nih.gov/32155444/) and likely [plays a role in SARS-CoV-2 pathogenesis](https://pubmed.ncbi.nlm.nih.gov/32869021/). This site is novel among SARS-like coronaviruses (sarbecoviruses), however, these sites [can be found](https://virological.org/t/naturally-occurring-indels-in-multiple-coronavirus-spikes/560) all across the coronavirus family, including the exact PRRAR (found in feline coronaviruses). When we wrote our paper, we predicted that related viruses may be found with insertions (if not necessarily polybasic sites) at the S1/S2 junction, and indeed [RmYN02](https://pubmed.ncbi.nlm.nih.gov/32416074/) - a SARS-like coronavirus found in bats - has an insertion at this junction. Similar to the SARS-CoV-2 RBD, the polybasic cleavage site is therefore a common feature observed during the evolution of coronaviruses. Update: there is now further evidence for this [observed in European bats](https://www.biorxiv.org/content/10.1101/2021.12.15.472779v1)

You can access the peer reviewed version of "Proximal Origin" from Nature Medicine [here](https://andersen-lab.com/papers/andersen-et-al-nature-medicine-2020/).

An earlier (non-peer reviewed) version also on [Virological](https://web.archive.org/web/20200217170645/https://virological.org/t/the-proximal-origin-of-sars-cov-2/398).

Very relevant (and technical) [Virological](https://virological.org/t/tackling-rumors-of-a-suspicious-origin-of-ncov2019/384) post by Bill Gallaher about the molecular mechanism of how SARS-CoV-2 gained its furin/polybasic cleavage site.

Another very relevant [Virological](https://virological.org/t/the-sarbecovirus-origin-of-sars-cov-2-s-furin-cleavage-site/536) post by Spyros Lytras about the evolution of the furin/polybasic cleavage site.

A detailed dissection of the evolution of furin cleavage sites across the coronavirus family from Bob Garry and Bill Gallaher on [Virological](https://virological.org/t/naturally-occurring-indels-in-multiple-coronavirus-spikes/560).

A followup study on [Virological](https://virological.org/t/spike-protein-sequences-of-cambodian-thai-and-japanese-bat-sarbecoviruses-provide-insights-into-the-natural-evolution-of-the-receptor-binding-domain-and-s1-s2-cleavage-site/622) looking at the many SARS-CoV-2 related viruses identified in diverse populations of bats, spread over large geographical areas.

#### Q&A about Fauci emails and early origin investigations
I did a Q&A with the New York Times to explain some of the key questions about my emails with Dr. Fauci and our early investigations into the origins of SARS-CoV-2. You can read the Q&A [here](https://www.nytimes.com/2021/06/14/science/covid-lab-leak-fauci-kristian-andersen.html). Update: Recent conspiracy theories have suggested that our scientific investigations were influenced by "politics" and "preferred narratives" from Drs. Fauci, Farrar, and others. Further, it has been suggested that "NIH leaders" (and others) provided edits to our papers. Such suggestions are false. Other conspiracy theories have suggested that we were awarded a grant from NIAID (our [WARN-ID center](https://warn-id.org/)) because of "preferred conclusions" in our Proximal Origin paper. Such suggestions are also false (and the grant was reviewed, scored, and ranked for funding in 2019 - prior to the pandemic).

#### Summary of the WHO report on the "Global Study of Origins of SARS-CoV-2"
I reanalyzed a lot of the data from the WHO report, which provided additional details of the early epidemic in Wuhan, including showing that the Huanan seafood market was the early epicenter and that SARS-CoV-2 was not widely circulating prior to the detection of the pandemic. I summarized all the main findings in this longer [Twitter thread](https://twitter.com/K_G_Andersen/status/1376954932004196352).

#### The SARS-CoV-2 furin cleavage site
It has been suggested that the furin cleavage site observed in SARS-CoV-2 is suggestive of the virus being engineered. As I lay out in [this Twitter thread](https://twitter.com/K_G_Andersen/status/1391507230848032772?s=20), that notion is incorrect.

#### Refuting the "Yan Report"
A [report](https://zenodo.org/record/4028830#.X1922Wj0nIW) from Dr. Li-Meng Yan and colleagues came out in September 2020 proposing that SARS-CoV-2 had been created in the lab. This report is one of many examples of conspiracy theories circulating about lab-based scenarios for the proximal origin of SARS-CoV-2. While such scenarios should not be dismissed without inquiry and deserve careful scientific inquiry (as we did in our "Proximal" paper), this particular report is entirely unsupported, non-scientific, and appear to have been created to support baseless conspiracy theories.

[Archived Twitter thread](https://web.archive.org/web/20210511050718/https://twitter.com/K_G_Andersen/status/1306037072914866178) outlining the issues with the Yan Report.

#### Authenticity of recent SARS-like coronavirus genomes
We downloaded raw data and reassembled relevant genomes from recently discovered SARS-like coronaviruses from bats and pangolins to assess their authenticity. The analyses show that all the genomes are authentic and as reported in the relevant manuscripts describing their sequencing and assembly.

Data available via [Google Cloud](https://console.cloud.google.com/storage/browser/andersen-lab_project_sars2-proximal-origins).

Summary of the findings with further discussion on [Virological](https://virological.org/t/on-the-veracity-of-ratg13/551).

In response to concerns raised to a paper published in PLOS Pathogens (Liu et al.), I performed a separate (and more in-depth) analysis for MP789 with the same results. Data and conclusion can be found via [Google Cloud](https://console.cloud.google.com/storage/browser/andersen-lab_project_sars2-proximal-origins/assemblies_MT121216?project=andersen-lab-primary).

#### Snakes not the reservoir of SARS-CoV-2
Early on in the pandemic it was suggested that snakes served as the reservoir of SARS-CoV-2. This conclusion was based on very naive and entirely wrong analyses. More in-depth analyses clearly show that snakes are unlikely to be the reservoir for the virus.

Post with analyses and discussion on [Virological](https://virological.org/t/ncov-2019-codon-usage-and-reservoir-not-snakes-v2/339).

Also very relevant post by David Robertson on [Virological](https://virological.org/t/ncovs-relationship-to-bat-coronaviruses-recombination-signals-no-snakes-no-evidence-the-2019-ncov-lineage-is-recombinant/331).

#### Early evolutionary estimates of SARS-CoV-2 molecular clock and the timing of the COVID-19 pandemic
In January 2020, we performed one of the earliest estimates of the evolutionary rate and timing of SARS-CoV-2 based on 27 sequences shared publicly on GISAID from Chinese investigators. The clock of ~1E-3 s/s/y and timing of mid-November / early-December have both remained remarkably stable as more genomes have been produced (more than 100k as of this writing).

Post on [Virological](https://virological.org/t/clock-and-tmrca-based-on-27-genomes/347) describing the findings, including important caveats and criticism of our analyses, etc.

---
**Andersen Lab**  
The Scripps Research Institute  
La Jolla, CA, USA  
[data@andersen-lab.com](mailto:data@andersen-lab.com)
